book/styles.md

```diff
@@ -264,7 +264,7 @@ So this is one way web pages can change their appearance. And in the
 early days of the web,^[I'm talking Netscape 3. The late 1990s.]
 something like this was the *only* way. But honestly, it's a
 pain---you need to set a `style` attribute on each element, and if you
-change the style, that's a lot of attributes to edit. CSS\index{CSS}
+redesign the page, that's a lot of attributes to edit. CSS\index{CSS}
```

Comment:
I think "change the style" doesn't make it obvious why it's more work than CSS since it could mean changing the style of just one element.